governance/tensorflow-testing.md

```diff
@@ -47,7 +47,7 @@ Once you have completed such a document, please email it to [Paige Bailey](mailt
 
 Between now and the preview launch for TensorFlow 2.0, we will be actively maintaining a discussion group for any questions, comments, suggestions, or issues that arise. **We will be holding a weekly stand-up for TF 2.0 testing via Hangouts** that will be announced through the TensorFlow Testing Discussion Group.
 
-_Please subscribe to [tf-2.0-testing@google.com](https://groups.google.com/a/google.com/forum/#!forum/tf-2.0-testing/new) to stay up-to-date._
+_Please subscribe to [tensorflow-testing@googlegroups.com](https://groups.google.com/forum/#!forum/tensorflow-testing) to stay up-to-date._
```

Comment:
Let's also home this file in SIG Build's community directory for now. We're not starting a new SIG just yet -- we're trialing the idea of a testing community, and I think Build is a reasonable place for us to put it.  You can move it to
https://github.com/tensorflow/community/tree/master/sigs/build

Reply:
(I don't want it in governance, because that's a place for permanent process docs.)